src/component/expense-form/expense-form.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
You're not exporting this component properly and that is a big reason why your app is breaking.